### PR TITLE
Updated Blog UI styles to match Marvel

### PR DESCRIPTION
**Pull request by CarterSimonson**
Teamwork: https://openforge.teamwork.com/#/tasks/16774931

### CODE REVIEW

**Comment by JaredKhalil** [comment]
Jared QA Check 12/27:

Blog(): If possible, make the 'What's New' blog article go away whenever the user: changes page, changes category, or uses the search function [this may need to be it's own separate task]
Blog(): More padding is needed between nav bar and top section of Blog

Mobile Only:
Blog(): No Search Bar on page yet

**Comment by luiskcs89** [comment]
Add this one: Blog(): If possible, make the 'What's New' blog article go away whenever the user: changes page, changes category, or uses the search function [this may need to be it's own separate task]
and for his one:
Mobile Only:
Blog(): No Search Bar on page yet
I do see the search bar
so I think it is ready for another round of QA

**Comment by Wholloran** [comment]
**All:** 
- The white space on both sides of the page should not be there (they are in the marvel but it's a weird thing that marvel does and shouldn't be incorporated)
- The social media icons open the site in the same tab, they should open in a new tab because they are external sites
- The links to peoples profiles "Jedi" and the links for "read more" should be orange, not blue
- Blog  page -> update the back button to just say "Back" per the marvel
- Blog  page -> The summary above the author should no longer be displayed https://marvelapp.com/57djie4/screen/60484467
- Blog  page -> the blog date should be next to the author's name
- Blog Page -> update the "Read next" section for each device type (the location of this area now differs for each device)


**Desktop / Tablet:**
- Desktop(): The width of the Tab bar w/ search bar is too wide see marvel for reference https://marvelapp.com/3f8d0id/screen/59791139

- Tablet(): There should not be a search bar in the tab bar just a magnifying glass. https://marvelapp.com/57djie4/screen/60484466
Clicking the magnifying glass should change the search bar to this view  https://marvelapp.com/57djie4/screen/63038794

- Tablet(): There is too much white space above the "Sign up for new" and the tab bar
- Blog page -> The Subscribe to the newsletter button is missing from the page 
- Blog  page -> The social media icons should be in line with the author's name now as well (see marvel) (Tablet/Desktop)
- increase the margins that the blog page is using right now the content is too skinny (the margins that the content in the current "read more" section are way better)

**Mobile:**
- The arrow from the tab bar is missing https://marvelapp.com/ifbe3gj/screen/60272333
- Update the styles for the "get the newsletter button & have it open the Mailchimp portal in another tab
- Remove the author image from displaying on mobile
- Move the social media icons to above "about the author" per the marvel https://marvelapp.com/ifbe3gj/screen/60272332
 - add back to blogs button to the bottom of the page

**Comment by Fdom92** [comment]
Added fix for:

- Update the styles for the "get the newsletter button & have it open the Mailchimp portal in another tab
- Remove the author image from displaying on mobile
- Move the social media icons to above "about the author" per the marvel https://marvelapp.com/ifbe3gj/screen/60272332
- add back to blogs button to the bottom of the page
- Blog page -> The social media icons should be in line with the author's name now as well (see marvel) (Tablet/Desktop)
- Tablet(): There is too much white space above the "Sign up for new" and the tab bar
- The links to peoples profiles "Jedi" and the links for "read more" should be orange, not blue
- Blog page -> update the back button to just say "Back" per the marvel
- Blog page -> The summary above the author should no longer be displayed
- Blog page -> the blog date should be next to the author's name
- Blog page -> The Subscribe to the newsletter button is missing from the page

**Comment by Fdom92** [comment]
Added fix for:

Tablet(): There should not be a search bar in the tab bar just a magnifying glass. https://marvelapp.com/57djie4/screen/60484466
Clicking the magnifying glass should change the search bar to this view https://marvelapp.com/57djie4/screen/63038794

**Comment by Wholloran** [comment]
- Change the Read next section. this section was redesigned and is now displayed differently on Mobile/tablet/desktop

Desktop:
- Add "get newsletter" button to the blog page https://marvelapp.com/3f8d0id/screen/59791144
- decrease the margin for the blog title and author so it's all inline. currently, they are using three different margins, per the marvel they should all use the same  https://marvelapp.com/3f8d0id/screen/59791144

Tablet:
- center the tab bar/search bar
- The "X" button to close the search bar does not have the "X" centered in it (vertically)
- The "get the newsletter" button on the blog article page doesn't do anything
- decrease the margin for the blog title, author, and image so it's all inline. currently, they are using three different margins, per the marvel they should all use the same https://marvelapp.com/57djie4/screen/60484467
- Author image is appearing as an oval rather than a circle 

Mobile:
- Remove the magnifying glass from the scrolling tab bar

**Comment by luiskcs89** [comment]
Ready, let me know if there are more tweaks needed and I'll switch to this as needed

**Comment by Wholloran** [comment]
Tablet:
- The "X" button to close the search bar does not have the "X" centered in it (vertically)
- Left Align the about the author section for tablet and widen the margins (to match main article width) (also the author image appears oval here)
- left align the "related blog articles" title and make the fon. a little larger

Mobile: There is white space on the sides of the feet on the blog page, there should not be

**Comment by luiskcs89** [comment]
Fixed

**Comment by Wholloran** [comment]
QA Passes

File: src/pages/app-blog/app-blog.scss [hunk 1]
```diff
@@ -124,52 +224,29 @@
...
-        color: #6b635f;
+        font-weight: 700;
+        color: #9baaab;
         cursor: pointer;
         text-transform: uppercase;
+        margin-bottom: 32px;
 
         @include media-breakpoint-down(md) {
-          padding: 26px 10px;
+          padding: 12px 10px;
```
Comment (PaulPaulDevelops) [inline src/pages/app-blog/app-blog.scss]: repetitive style declarations when declaring `padding` then `padding-left` `padding-right` etc.

File: src/pages/app-blog/app-blog.scss [hunk 2]
```diff
@@ -24,98 +29,193 @@
...
+        margin-top: 0;
+        margin-left: 0;
+      }
+    }
+
+    .blog-search-icon-top {
+      position: absolute;
+      margin-top: 10px;
+      margin-left: 10px;
+      color: #615853;
```
Comment (PaulPaulDevelops) [inline src/pages/app-blog/app-blog.scss]: Do we have a global variable for this color? If not, we should probably add it for ease of use. I've seen it a few times within this file.

File: src/pages/app-blog/app-blog.scss [hunk 3]
```diff
@@ -24,98 +29,193 @@
...
+      background-color: #f4f5f9;
+      border-radius: 3px;
+      width: 85%;
+      height: 32px;
+      font-family: Muli;
+      color: #5d534e;
+      font-size: 13px;
+      font-weight: 300;
+      line-height: 18px;
+      border: 1.5px solid rgba(0, 0, 0, 0.15);
```
Comment (PaulPaulDevelops) [inline src/pages/app-blog/app-blog.scss]: Border styles are set twice within this class.

File: src/pages/app-blog/app-blog.scss [hunk 4]
```diff
@@ -24,98 +29,193 @@
...
+      background-color: #f4f5f9;
+      box-shadow: inset -1px -1px 1px 0 rgba(0, 0, 0, 0.15);
+      border-radius: 3px;
+      width: 100%;
+      height: 32px;
+      font-family: Muli;
+      color: #5d534e;
+      font-size: 13px;
+      font-weight: 300;
+      line-height: 18px;
```
Comment (PaulPaulDevelops) [inline src/pages/app-blog/app-blog.scss]: Not functionality related, but for longer styles like this example I like putting the styles in alphabetical order. It's easier to view top down when you can view all the border styles together, all the font styles together, etc.

File: src/pages/app-blog/app-blog.scss [hunk 5]
```diff
@@ -24,98 +29,193 @@
...
 
-      input {
-        border-radius: 2px;
-        box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.27);
-        background-color: #fff;
-        border: 0;
-        padding: 5px;
-      }
+    input {
+      border-radius: 2px;
```
Comment (PaulPaulDevelops) [inline src/pages/app-blog/app-blog.scss]: is `border-radius` still valid with the bottom styles?

File: src/components/app-blog-content/app-blog-content.scss [hunk 6]
```diff
@@ -104,22 +104,102 @@
...
+  display: none;
+
+  @include media-breakpoint-down(md) {
+    display: block;
+  }
+}
+
+.blog-next-posts {
+  h1 {
+    color: #292a2d;
```
Comment (PaulPaulDevelops) [inline src/components/app-blog-content/app-blog-content.scss]: Do we have a global variable for this color? If not, we should probably add it for ease of use. I've seen it a few times within this file.

**Comment by luiskcs89** [comment]
syles refactored

**Pull request merged**